website/docs/plugins/ax_sweeper.md

```diff
@@ -0,0 +1,55 @@
+---
+id: ax_sweeper
+title: Ax Sweeper plugin
+sidebar_label: Ax Sweeper plugin
+---
+This plugin provides a mechanism for Hydra applications to use the <a class="external" href="https://ax.dev/" target="_blank">Adaptive Experimentation Platform, aka Ax</a>. Ax can optimize any experiment - machine learning experiments, A/B tests, and simulations.
+
+Install with 
+```
+pip install hydra_ax_sweeper
+```
+
+Once installed, override `hydra/sweeper` in your config:
+
+```yaml
+defaults:
+  - hydra/sweeper: ax
+```
```

Comment:
mention the general configuration options (early stop, minimize/maximize etc).
(Also in README if you didn't).